include/vcpkg/base/sortedvector.h

```diff
@@ -1,36 +1,56 @@
 #pragma once
 
 #include <algorithm>
+#include <functional>
+#include <initializer_list>
+#include <iterator>
 #include <vector>
 
 // Add more forwarding functions to the m_data std::vector as needed.
 namespace vcpkg
 {
-    template<class T>
+    template<class Ty>
```

Comment:
I'm sorry, I'd prefer the old version that just takes a `Compare`; I care more about the API here than the extra eight bytes.